docs/api/shell.md

```diff
@@ -20,13 +20,13 @@ The `shell` module has the following methods:
 
 * `fullPath` String
 
-Show the given file in a file manager. If possible, select the file.
+Show the given file in a file manager. If possible, select the file. Expects `fullPath` to be a fully-specified path (`/users/<username>/path/to/file` or `~/path/to/file`). Relative paths (`./file`) do not work.
```

Comment:
I just tested and `~` does not appear to work with Electron on Windows 8.1. I am doubtful it would work on other Windows versions. I believe the tilde assignment is Unix specific opposed to being an Electron bug.

Interestingly, .Net projects can use `~` in Visual Studio and `~` is also available in Powershell but not CMD. So I take it the usage in Windows is implemented on an application level and not an OS level.